lib/membership/index.js

```diff
@@ -199,11 +199,66 @@ Membership.prototype.isPingable = function isPingable(member) {
             Member.isStatusPingable(member.status);
 };
 
+/**
+ * Change the status of the local member to alive
+ * @see Membership#setLocalStatus
+ * @see Member.Status.alive
+ */
+Membership.prototype.makeLocalAlive = function makeLocalAlive(){
+    this.setLocalStatus(Member.Status.alive);
+};
+
+/**
+ * Bump the incarnation number of the local member and return an update that
+ * described the current state of the local member.
+ *
+ * @returns {Update} The update to gossip around
+ * @private
+ */
+Membership.prototype._reincarnate = function _reincarnate() {
+    this.localMember.incarnationNumber = Date.now();
+
+    return new Update(this.localMember.address, this.localMember.incarnationNumber, this.localMember.status, this.localMember);
```

Comment:
With the changes in mind we want to make later does it make sense to refactor the `new Update(...)` call to take two members instead of all fields separated? We can also hold of to a later state when we actually extend the membership state with labels.